src/azul/service/catalog_controller.py

```diff
@@ -55,13 +63,18 @@ def list_catalogs(self) -> schema.object(
         }
 
     @cache
-    def _plugin_config(self, plugin_base_cls: str, catalog: str):
+    def _plugin_config(self, plugin_base_cls: str, catalog: CatalogName) -> JSON:
         plugin_base_cls = Plugin.type_for_name(plugin_base_cls)
+        plugin_cls = plugin_base_cls.load(catalog)
         if issubclass(plugin_base_cls, RepositoryPlugin):
-            plugin_cls = plugin_base_cls.load(catalog)
             plugin = plugin_cls.create(catalog)
             return {
                 'sources': list(map(str, plugin.sources))
             }
+        elif issubclass(plugin_base_cls, MetadataPlugin):
+            plugin = plugin_cls.create()
+            return {
+                'indices': list(map(attr.asdict, plugin.indices().values()))
```

Comment:
```suggestion
                'entity_types': list(map(attr.asdict, plugin.indices().values()))
```

For the same reason as above.